examples/apps/data-object-grid/package.json

```diff
@@ -30,7 +30,6 @@
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
 		"prepack": "npm run webpack",
 		"start": "webpack serve",
-		"start:server": "tinylicious",
```

Comment:
To prevent this in the future, add `["tinylicious", "tinylicious"]` to the policy config:

https://github.com/microsoft/FluidFramework/blob/b3bfc6ad76a2294e2520b5221d645e24960a5c94/fluidBuild.config.cjs#L408-L429

Reply:
Good idea, added.